content/influxdb/v2.0/process-data/common-tasks/calculate_weekly_mean.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Calculate a weekly mean
+description: >
+  Calculate a weekly mean and add it to a new bucket.
+menu:
+  influxdb_2_0:
+    name: Calculate a weekly mean
+    parent: Common tasks
+weight: 202
+influxdb/v2.0/tags: [tasks]
+---
+
+{{% note %}}
+This example uses [NOAA water sample data](/influxdb/v2.0/reference/sample-data/#noaa-water-sample-data).
+{{% /note %}}
+
+Calculate a weekly mean and store it in a separate bucket.
+
+This example groups average temperature by week and computes the mean using the [`aggregateWindow()` function](/influxdb/v2.0/reference/flux/stdlib/built-in/transformations/aggregates/aggregatewindow/), then sends the weekly mean to a new bucket (`weekly_means`).
+
+This example uses [NOAA water database data](https://influx-testdata.s3.amazonaws.com/noaa.csv) and the experimental [`csv.from()` function](/influxdb/v2.0/reference/flux/stdlib/experimental/csv/from/) to retrieve data used to calculate the weekly mean.
+
+```js
+import "experimental/csv"
+
```

Comment:
You can delete the import statement since you no longer need the experimental csv package.